src/ServerBlock.hpp

```diff
@@ -3,6 +3,26 @@
 
 #include "webserv.hpp"
 
+// a list of all accepted variables
+// ---------------------------------
+// 1. listen, 2. server_name, 3. error_page,
+// 4. index, 5. root, 6. client_max_body_size, 7. autoindex, 
+// 8. allow_methods, 9. alias, (cgi_path, cgi_ext) - optional
+// ---------------------------------
+struct Variables
```

Comment:
Can we change Variables to Directives?

Reply:
It is hard to watch how variables are being constantly discriminated in this team...
OK!